streamflow/core/utils.py

```diff
@@ -49,6 +50,10 @@ def __contains__(self, name: str) -> bool:
         return False
 
 
+def contains_persistent_id(id_: int, entities: Iterable[PersistableEntity]) -> bool:
```

Comment:
This function seems to be used only in the `tests` package. Better moving it to the `tests.utils.utils` module